atomicapp/providers/openshift.py

```diff
@@ -36,91 +35,205 @@ class OpenShiftProvider(Provider):
     cli = None
     config_file = None
     template_data = None
+    providerapi = "https://127.0.0.1:8443"
```

Comment:
Ha, `constant.py` was my first thought too, but it's provider specific thing and as providers are plugins, they should be removable without leaving traces behind (like unused constants etc.)